tests/Unit/Helper/Cron/UtilityTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+namespace Shopgate\Export\Test\Unit\Helper\Cron;
+
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Sales\Model\Order;
+use Shopgate\Export\Helper\Cron\Utility as CronHelper;
+
+/**
+ * @coversDefaultClass \Shopgate\Export\Helper\Product\Utility
+ */
+class UtiltiyTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var ObjectManager */
+    private $objectManager;
+    /** @var CronHelper */
+    private $cronHelper;
+
+    /**
+     * Load object manager for initialization
+     */
+    public function setUp()
+    {
+        $this->objectManager = new ObjectManager($this);
+        /** @var \Shopgate\Export\Helper\Cron\Utility $cronHelper */
+        $this->cronHelper = $this->objectManager->getObject(
+            'Shopgate\Export\Helper\Cron\Utility'
```

Comment:
use `Shopgate\Export\Helper\Cron\Utility::class` to derive the class instead of string

Reply:
changed